plsync/sites.py

```diff
@@ -131,6 +131,7 @@
     makesite('mil02','80.239.222.0',   '2001:2030:30::',       'Milan', 'IT', 45.464000, 9.191600, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
     makesite('mil03','77.67.115.0',    '2001:668:1f:62::',     'Milan', 'IT', 45.464000, 9.191600, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
     makesite('mnl01','202.90.156.0',   '2001:d18:0:35::',      'Manila', 'PH', 14.5086, 121.0194, user_list, count=4, arch='x86_64', nodegroup='MeasurementLabCentos'),
+    makesite('mpm01','41.94.23.0',   None,      'Maputo', 'MZ', -21.7033409, 25.149247, user_list, count=4, arch='x86_64', nodegroup='MeasurementLabCentos'),
```

Comment:
Hi Chris,

A couple comments:

* The coordinates you have [appear to be the in the middle of nowhere Botswana](https://www.google.com/maps/place/21%C2%B042'12.0%22S+25%C2%B008'57.3%22E/@-23.0495212,19.9368781,5z/data=!4m5!3m4!1s0x0:0x0!8m2!3d-21.7033409!4d25.149247). As [far as I can tell](http://www.aeroportos.co.mz/por/Aeroportos/Maputo), the [coordinates](https://en.wikipedia.org/wiki/Maputo_International_Airport) should be `-25.9208, 32.5725`.
* Since this site has no IPv6, I believe you are going to need to add `exclude=[1,2,3,4]`.
* Could you indent the items to match the columns of the line above it?

Thanks!

Nathan